flaml/default/suggest.py

```diff
@@ -3,21 +3,28 @@
 import logging
 import pathlib
 import json
-from flaml.data import CLASSIFICATION, DataTransformer
+from flaml.data import CLASSIFICATION, DataTransformer, _is_nlp_task
 from flaml.ml import get_estimator_class, get_classification_objective
 
 LOCATION = pathlib.Path(__file__).parent.resolve()
 logger = logging.getLogger(__name__)
 CONFIG_PREDICTORS = {}
 
 
-def meta_feature(task, X_train, y_train):
-    is_classification = task in CLASSIFICATION
-    n_row = X_train.shape[0]
-    n_feat = X_train.shape[1]
-    n_class = len(np.unique(y_train)) if is_classification else 0
-    percent_num = X_train.select_dtypes(include=np.number).shape[1] / n_feat
-    return (n_row, n_feat, n_class, percent_num)
+def meta_feature(task, estimator_class, X_train, y_train, strategy="data_size"):

```

Comment:
```suggestion
def meta_feature(task, X_train, y_train, meta_feature_names):
```
"data_size" is not used in the current code (with test coverage). If you want to keep it, need to add comments to explain what it's used for and use it in code and make all code paths covered.
As for why I use "meta_feataure_names", see my comment below in `suggest_config`.